### DISCUSSION TITLE
item list for call

### DISCUSSION THREAD

**Discussion by Peter-Jacob** [adesutherland/CREXX#73] [General]
- on which branch should we develop? it seems René and I are on different ones. Therefore his substr stuff doesn't work on mine (f28)  
- is pushing changes to GIT still the way to release changes/amendments? Are bifs also part of it? 
- bifs, how are they defined, compiled, and used
- which super instructions should we implement, naming conventions
- development in rexx asm versus rexx. Is tiny rexx still an option.

**Comment by rvjansen** [comment]
I am on f0028 from yesterday. Built on three different machines so chance of incidental versioning mishaps is low.
